recipes/server_check_helper.rb

```diff
@@ -0,0 +1,23 @@
+
+require 'ipaddr'
+
+def is_server_disallowed?(retry_join_list, my_ip)
```

Comment:
Personal preference, what do you think?

```suggestion
def is_server_allowed?(retry_join_list, my_ip)
```

Then we'd have

```ruby
if !is_server_allowed?
  # disable consul
```

It works better with `'allowed' => true` suggested below in tests.

Reply:
Yeah, agree with the above point from @glynnforrest . A double negative is confusing.

Reply:
Fixed in Fixed in https://github.com/Roblox/consul-cookbook/pull/44/commits/a3210fa5d0d7047e244e6446f47c90bc1ccd1a90